src/test/Components/Blueprints/Blueprints.test.tsx

```diff
@@ -203,7 +203,7 @@ describe('Blueprints', () => {
         {},
         routes
       );
-      const blueprintDetails = await screen.findByText('Image details');
+      const blueprintDetails = await screen.findByText('Details');
       await user.click(blueprintDetails);
       await screen.findByText(editedBlueprintName);
```

Comment:
"Details" appears twice on the Review step, once in a left side navigation and once as a heading of the expandable. This should do the trick :)

```suggestion
      const blueprintDetails = await screen.findAllByText('Details');
      await user.click(blueprintDetails[1]);
      await screen.findByText(editedBlueprintName);
```

Reply:
Or we can use `image-details-expandable` testId. Which might be better because then there's no need to "guess" the index of the correct "Details"

```suggestion
      const blueprintDetails = await screen.findByTestId(
        'image-details-expandable'
      );
      await user.click(blueprintDetails);
      await screen.findByText(editedBlueprintName);
```